src/atomicdex/app.cpp

```diff
@@ -232,11 +152,9 @@ namespace atomic_dex
         this->process_one_frame();
         if (m_event_actions[events_action::need_a_full_refresh_of_mm2])
         {
-            auto& mm2_s = system_manager_.create_system<mm2_service>(system_manager_);
-
-            system_manager_.create_system<coinpaprika_provider>(mm2_s);
-            // system_manager_.create_system<ohlc_provider>(mm2_s);
+            system_manager_.create_system<mm2_service>(system_manager_);
```

Comment:
Would be nice to have a global refactor in antara-gaming-sdk using `m_` as prefix instead of `field_`